src/index.md

```diff
@@ -1,7 +1,8 @@
 ---
 layout: layouts/main.njk
 title: Home
-background: ./src/img/background/home.jpg
+background: ./src/img/background/home-light.jpg
+darkBackground: ./src/img/background/home.png
```

Comment:
you should make this image a compressed jpg instead, a 8MB png is really big for a website image

Reply:
Use the WEBP format instead.

Reply:
was suggesting jpg for consistency, but yeah webp is even better